pkg/chartverifier/checks/charttesting.go

```diff
@@ -4,13 +4,43 @@ import (
 	"fmt"
 	"strings"
 
+	"github.com/Masterminds/semver"
 	"github.com/helm/chart-testing/v3/pkg/chart"
 	"github.com/helm/chart-testing/v3/pkg/config"
 	"github.com/helm/chart-testing/v3/pkg/exec"
 	"github.com/helm/chart-testing/v3/pkg/util"
 	"github.com/redhat-certification/chart-verifier/pkg/tool"
 )
 
+// Versioner provides OpenShift version
+type Versioner interface {
+	getVersion() (string, error)
+}
```

Comment:
Versioner could be `type Versioner func() (string, error)` instead; this makes it easier later on when overriding it in-place.

Reply:
Done, My first foray into function type so hopefully I did it correctly - at least the tests pass.